backend/src/main/java/com/MerchStore/backend/Model/Product.java

```diff
@@ -10,10 +10,10 @@ public class Product {
 
     private double price;
 
-    private int stock;
+    private long stock;
```

Comment:
long is an overkill for stock too, i hardly doubt we would have large inventory.

Reply:
I have changed the data type of stock to int.